### DISCUSSION TITLE
How secure can we say that flet is today, and the pretense for future versions will be big changes in the API currently?

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#1369] [Q&A]
Feodor, are you all right?
I can say that I am in love with the flet.
I want to use it in production in a larger project, but I have some questions:
- In terms of security, how safe is it to use flet today?
- Currently flet is in version 0.6.2, with this version, will I be able to update the version easily without my project breaking?

**Comment by lawchin** [comment]
all I can say is.... flet gets update quite fast compare to pynecone and kivy.

**Comment by abcnow** [comment]
@nascin were you able to use flet in production? Share your experience if any? thanks in advance!

**Reply by FeodorFitsner** [reply]
Hey, thanks for your interest!

We are still updating roadmap for 2023-24, but right now I can tell that we are going to dedicate the rest of this year to stabilization work.

For example, right now I'm working on Flet integration into FastAPI server for production web workloads.

What else concerns you might have that I can address?

**Reply by abcnow** [reply]
@FeodorFitsner thanks a lot for the update! That's exciting man! Fastapi seems to be the perfect combination! I can't wait to see that.. 
I am also looking forward to seeing database integration that is on the roadmap for this year..
